sdks/python/apache_beam/io/hadoopfilesystem_test.py

```diff
@@ -0,0 +1,464 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Unit tests for HadoopFileSystem."""
```

Comment:
<!--thread_id:cc_157939392_t; commit:5e9211de2f83b679cfc3ea9e6769b1f22ac0215c; resolved:1-->
<!--section:context-quote-->
> **chamikaramj** wrote:
> \`\`HadoopFileSystem\`\`

<!--section:body-->
We don't generate docs for test files.